packages/strapi-plugin-users-permissions/services/UsersPermissions.js

```diff
@@ -231,4 +231,97 @@ module.exports = {
       strapi.log.error(err);
     }
   },
+
+  syncSchema: (cb) => {
+    const Model = strapi.plugins['users-permissions'].models.user;
+
+    if (Model.orm === 'bookshelf') {
```

Comment:
A better pattern for this big condition, could be make the stop before.

```js
if (Model.orm !== 'bookshelf') {
  return cb();
}
```

Then, you don't need to embrace all the business logic into the conditional if/else.